pkg/logs/input/kubernetes/launcher.go

```diff
@@ -144,32 +144,57 @@ func (l *Launcher) removeSource(service *service.Service) {
 	}
 }
 
-// kubernetesIntegration represents the name of the integration.
-const kubernetesIntegration = "kubernetes"
+const (
+	// kubernetesIntegration represents the name of the integration.
+	kubernetesIntegration   = "kubernetes"
+	podStandardLabelPrefix  = "tags.datadoghq.com/"
+	tagKeyService           = "service"
+	podStandardLabelService = podStandardLabelPrefix + tagKeyService
```

Comment:
we probably need to import them from the tagger pkg instead of redefining them here

Reply:
for some reason was having trouble with this, whats the correct way to import?  I had added the collectors (and tried taggers) package to the import statement but still wasn't able to access those variables - and do they need to be capitalized to export?